tests/simpleTest/simple_test_replica.hpp

```diff
@@ -19,7 +19,9 @@
 #include "Replica.hpp"
 #include "ReplicaConfig.hpp"
 #include "ControlStateManager.hpp"
```

Comment:
move the implementation in a separate .cpp file and keep the definitions in .hpp